app/screens/settings/settings-screen.tsx

```diff
@@ -83,6 +103,10 @@ export const SettingsScreen = observer(function SettingsScreen({ navigation }: S
         <SettingBox last title={translate("settings.about")} icon="ℹ️" chevron onPress={() => navigation.navigate("about")} />
       </View>
 
+      <View style={SETTING_GROUP}>
+        <SettingBox first last title={"Delete Data"} icon="🗑️" onPress={handleDeleteAllData} />
```

Comment:
Can you please move this box to the Setting's privacy screen (`PrivacyScreen`)? It can be found inside the About section